.github/workflows/build-wasm.yml

```diff
@@ -0,0 +1,38 @@
+name: Build wasm app
+
+on: push
+
+jobs:
+  buster:
+    name: wasm build
+    runs-on: ubuntu-latest
+    container:
+      image: archlinux:latest
```

Comment:
you should be able to just replace archlinux:latest by qt-wasm:latest here actually